### ISSUE TITLE
ContinuationToken exposure for DeviceTwin

### ISSUE THREAD

**Issue by zhang-hua** [Azure/iothub-manager-java#28]
# Type of issue

- [ ] Bug
- [x] New feature
- [] Enhancement

# Description

IotHub Java SDK  latest version 1.9.25 doesn't expose continuationToken functions of device twin query to the client. You can only use DeviceTwin methods hasNextDeviceTwin and getNextDeviceTwin to query device twins. The request and response continuationToken are private members encapsulated in Query. This is not equivalent to IotHub dotnet SDK. We need the Java SDK to expose this functionality to the client.

The requires implementation of Azure IoT Java SDK. An issue has been created at https://github.com/Azure/azure-iot-sdk-java/issues/145.

# Steps to reproduce

1. [First step]
2. [Second step]
3. [and so on...]

# Expected behavior

...

# Current behavior

...

# Known workarounds

...

# Possible solution

...

# Context and Environment
* Operating System: ...
* GitHub branch: ...
* Java JDK/JRE: ...

# Code to reproduce the bug

```
import org.junit.*;
import org.junit.experimental.categories.Category;

public class MyTest {

    @Test(timeout = 5000)
    public void TestToReproduceIssue() {
        // Arrange
        ...

        // Act
        ...

        // Assert
        ...
    }
}
```

**Comment by zhang-hua** [comment]
The Java SDK has exposed this support in new release [2017-10-23](https://github.com/Azure/azure-iot-sdk-java/releases/tag/2017-10-23). But this feature is not used by web UI for now.

**Comment by zhang-hua** [comment]
Need to remove the work around code once we bump Java SDK to the new release which include fix for issue [176](https://github.com/Azure/azure-iot-sdk-java/issues/176).

**Comment by zhang-hua** [comment]
SDK issue [176](https://github.com/Azure/azure-iot-sdk-java/issues/176) has been fixed in release 2017-12-2.  This issue can be closed once PR #82 is merged.

**Comment by zhang-hua** [comment]
close it since all related issues has been fixed.
